lib/repos/game.rb

```diff
@@ -17,7 +17,7 @@ def initialize(ipfs_addr)
         @signer=@initblock.signer
```

Comment:
sorry just explaining the reasoning behind the next commit, all this processing of the initial block should happen inside the game managing thing, this guys job is just to get the JSON and send it to the right place